src/nodes/pmrem/PMREMNode.js

```diff
@@ -203,6 +203,7 @@ class PMREMNode extends TempNode {
 	/**
 	 * The node's texture value.
 	 *
+	 * @param {Texture} value
```

Comment:
I don't think this bit is right. We do not document a setter here but just a property. Properties don't use `param`.

Reply:
Well I was thinking the same at first but without this line, eslint hrow this `Missing JSDoc for parameter 'value'.eslint[valid-jsdoc]` 🤔

Reply:
Seems like some recommendations are to not document the setter at all, what do you think ? https://github.com/jsdoc/jsdoc/issues/973

Reply:
The idea is to document setter/getter only once per name.

In this case I would remove the ESLint rule.

Reply:
Yes, the documentation is also available on the setter 
![image](https://github.com/user-attachments/assets/881e993b-7cba-42e0-ac81-ba2f1e336642)